.meta/_partials/fields/_includes_excludes.toml

```diff
@@ -0,0 +1,22 @@
+[<%= namespace %>]
+type = "table"
+category = "Filesystem"
+description = "Lists of <%= term %> patterns to include or exclude."
+
+[<%= namespace %>.options.includes]
+type = "[string]"
+required = false
+examples = <%= examples.to_toml %>
+description = """\
+The list of <%= term %> patterns for which to gather <%= metrics %> metrics. \
+Defaults to including all <%= term %>s.\
```

Comment:
I guess. My thinking was that making them options skips a couple steps of indirection in the common case (no iterator, no call to fnmatch). However, it does complicate the matching code, and handling them is not a hot path (unless the system has a huge number of devices, but even then it is polled at most once a second). I'll switch.

Reply:
Ah, there's a gotcha here. For filesystem metrics, the filesystem may have no device. If no patterns are specified, this filesystem currently would be included, but any device pattern would allow the user to chose to exclude them. Forcing a pattern match would require us to either always include or always exclude such filesystems. Now, these filesystems are *probably* not interesting for metrics, but it is a consideration.

Reply:
:thinking: I think I'd expect "no device" to also match `*` though so, if I literally specified `[ "*" ]` as the config, I'd still get a match. I concede that is a bit of an odd case.